ariadne/cardano/src/Ariadne/Wallet/Cardano/WalletLayer/Types.hs

```diff
@@ -83,6 +83,8 @@ data PassiveWalletLayer m = PassiveWalletLayer
     , pwlRollbackBlocks
           :: NewestFirst NE Blund
           -> m ()
+    , pwlGetPassiveWallet
```

Comment:
I don't think it's a good idea to expose `PassiveWallet`. Why not add more specific function `pwlCleanupDBPeriodically` instead?

Reply:
Not sure if `periodically` should be there (do we want to have Kernel manage its own threads?). `pwlCleanupAcidState` seems like a better idea.